python/cudf/cudf/_lib/string_casting.pyx

```diff
@@ -771,3 +777,48 @@ def is_hex(Column source_strings):
         ))
 
     return Column.from_unique_ptr(move(c_result))
+
+
+def from_decimal(Column input_col):
+    cdef column_view input_column_view = input_col.view()
+    cdef unique_ptr[column] c_result
+    with nogil:
+        c_result = move(
+            cpp_from_fixed_point(
+                input_column_view))
+
+    return Column.from_unique_ptr(move(c_result))
+
+
+def to_decimal(Column input_col, object out_type):
+    cdef column_view input_column_view = input_col.view()
+    cdef unique_ptr[column] c_result
+    cdef int scale = out_type.scale
+    cdef data_type c_out_type = data_type(DECIMAL64, -scale)
+    with nogil:
+        c_result = move(
+            cpp_to_fixed_point(
+                input_column_view,
+                c_out_type))
+
+    result = Column.from_unique_ptr(move(c_result))
+    result.dtype.precision = out_type.precision
+    return result
+
+
+def is_fixed_point(Column input_col, object out_type):
```

Comment:
> Is out_type an appropriate name for the dtype here? Should we use just dtype instead?

It was intended to match with the `to_decimal` parameter above since it basically checks the sanity of the data that `to_decimal` would create. 
I don't mind changing it.